biogtr/io/association_matrix.py

```diff
@@ -211,8 +260,10 @@ def __getitem__(self, inds) -> np.ndarray:
 
         query_ind = self.__getindices__(query_inst, self.query_instances)
         ref_ind = self.__getindices__(ref_inst, self.ref_instances)
+
         try:
             return self.numpy()[query_ind[:, None], ref_ind].squeeze()
+
         except IndexError as e:
             print(f"Query_insts: {type(query_inst)}")
             print(f"Query_inds: {query_ind}")
```

Comment:
This is debugging code and comes at the cost of added checks and try-except fencing. I propose just getting rid of it. It's easier to debug these types of things now that things are better refactored.